packages/geo/src/lib/datasource/shared/datasources/wfs.service.ts

```diff
@@ -226,10 +153,11 @@ export class WFSService extends DataService {
       let fieldListWoGeom;
       let fieldListWoGeomStr;
       let olFormats;
-      const patternGml3 = /gml/gi;
-      if (wfsDataSourceOptions.paramsWFS.outputFormat.match(patternGml3)) {
+      const outputFormat = wfsDataSourceOptions.paramsWFS.outputFormat;
+
+      if (gmlRegex.test(outputFormat) || !outputFormat) {
         olFormats = olformat.WFS;
-      } else {
+     } else {
         olFormats = olformat.GeoJSON;
       }
 
```

Comment:
I think this would be a fine case for the concatMap or zip operators.

Reply:
Code has been simplified. This is no longer applicable.

Reply:
Code has been simplified. This is no longer applicable.